policyengine_taxsim/config/variable_mappings.yaml

```diff
@@ -38,250 +54,412 @@ policyengine_to_taxsim:
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "17. Deductions allowed \n  QBI deduction"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v18:
     variable: taxable_income
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "18. Federal Taxable Income"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v19:
     variable: income_tax_before_credits
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "19. Federal Regular Tax"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v20:
     variable: not_implemented_in_pe
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "20. Exemption Surtax 1988-96"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v21:
     variable: not_implemented_in_pe
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "21. General Tax Credit 1975-8"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v22:
     variable: ctc
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "22. Child Tax Credit*17/22 98"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v23:
     variable: refundable_ctc
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "23. Refundable Part"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v24:
     variable: cdcc
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "24. Child Care Credit 1076+"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v25:
     variable: eitc
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "25. Earned Income Credit 1975"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v26:
     variable: amt_income
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "26. Alternative Min Income"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v27:
     variable: alternative_minimum_tax
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "27. AMT"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v28:
     variable: income_tax_before_refundable_credits
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "28. Income Tax Before Credits \n Total Credits"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v29:
```

Comment:
Same as V17:
should be 2 outputs:

We were able to generate an output previously - not sure which variable it was - will reach out to Martin

`taxsim_tfica` - this is the taxpayer share